tests/Aspire.Qdrant.Client.Tests/ConformanceTests.cs

```diff
@@ -37,17 +50,38 @@ protected override void RegisterComponent(HostApplicationBuilder builder, Action
     }
 
     protected override void PopulateConfiguration(ConfigurationManager configuration, string? key = null)
-        => configuration.AddInMemoryCollection(new KeyValuePair<string, string?>[2]
+    {
+        configuration.AddInMemoryCollection(
+            [
+             new KeyValuePair<string, string?>(CreateConfigKey("Aspire:Qdrant:Client", key, "Endpoint"), GetConnectionStringKeyValue(_connectionString,"Endpoint")),
+             new KeyValuePair<string, string?>(CreateConfigKey("Aspire:Qdrant:Client", key, "Key"), GetConnectionStringKeyValue(_connectionString,"Key")),
+             new KeyValuePair<string, string?>($"ConnectionStrings:{key}",_connectionString)
+            ]);
```

Comment:
You shouldn't need to set the `ConnectionStrings` config value. You should just need to set `Endpoint` and `Key`.